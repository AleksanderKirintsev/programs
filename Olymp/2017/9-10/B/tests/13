10
11 100
24 66
87 97
61 67
13 90
95 95
81 81
10 71
16 96
47 49
