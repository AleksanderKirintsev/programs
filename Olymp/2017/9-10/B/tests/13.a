72 100
-1 -1
-1 -1
-1 -1
-1 -1
-1 -1
-1 -1
10 71
-1 -1
-1 -1
