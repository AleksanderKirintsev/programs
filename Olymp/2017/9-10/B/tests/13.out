97 100
-1 -1
97 97
-1 -1
-1 -1
-1 -1
-1 -1
10 71
72 96
-1 -1
